### DISCUSSION TITLE
no Overall Power, only Feed in R-S-T and Grid Consumption R-S-T; H3 Inverter

### DISCUSSION THREAD

**Discussion by horstvogel** [nathanmarlor/foxess_modbus#633] [General]
Hello, there is no "overall power" in the Fox Ess Modbus integration. 

I Cloud only find Feed in R-S-T and Grid Consumption R-S-T, not the not the accumulated sum of these values. 

This would be interesting for zero feed-in

thanks.

**Comment by FozzieUK** [comment]
There’s been a lot of discussion on this which you can find on this thread, have a look at this to see if it helps https://github.com/nathanmarlor/foxess_modbus/discussions/456 , this one https://github.com/nathanmarlor/foxess_modbus/discussions/583  and this one https://github.com/nathanmarlor/foxess_modbus/discussions/462 - I think most 3 phase users have built the various sensors needed to calculate combined power from the sums of R-S-T
